flow-server/src/test/java/com/vaadin/flow/server/communication/StreamRequestHandlerTest.java

```diff
@@ -143,6 +146,44 @@ public void stateNodeStates_handlerMustNotReplyWhenNodeDisabled()
         Mockito.verify(response).sendError(403, "Resource not available");
     }
 
+    @Test
+    public void stateNodeStates_handlerMustReplyWhenNodeDisabledButAllowsAllMode()
```

Comment:
```suggestion
    public void nodeDisabled_shouldReplyForDisabledUpdateModeAlways()
```

the beginning stateNodeStates doesn't give insight to what the test tests, should be more descriptive only of what is tested and not have a test collection name at the start.